apps/src/aichat/views/ChatMessageView.tsx

```diff
@@ -88,26 +88,30 @@ const ChatMessageView: React.FunctionComponent<ChatMessageViewProps> = ({
       ) : null;
   }
 
-  if (!isAssistant && !isChatHistoryView && assets && currentChannelId) {
-    footer = (
-      <div className={styles.assetRow}>
+  let header;
+  if (!isAssistant && assets && currentChannelId) {
+    header = (
+      <div className={styles.assetCol}>
         {assets.map(asset => {
           const filename = asset.filename;
-          return filename.endsWith('.pdf') ? (
-            <div key={filename} className={styles.pdfPreview}>
-              <FontAwesomeV6Icon
-                iconName="file-pdf"
-                className={styles.pdfIcon}
-              />
-              <span>{filename}</span>
-            </div>
-          ) : (
-            <img
-              key={filename}
-              alt=""
-              className={styles.imagePreview}
-              src={getAssetUrl(asset, currentChannelId, levelName)}
-            />
+          const url = getAssetUrl(asset, currentChannelId, levelName);
+          return (
+            <button
+              type="button"
+              className={styles.assetButton}
+              onClick={() => window.open(url, '_blank')}
+            >
+              {filename.endsWith('.pdf') ? (
+                <FilePreview type="pdf" filename={filename} url={url} />
+              ) : (
+                <img
```

Comment:
Could this be handled in `FilePreview` as well with different CSS classnames?

Reply:
Yes probably, seemed simple enough as-is but can try and put it all in one place

Reply:
I tried, and I think it's a bit of a PITA to do this (ie, we'd need to override the border radius styles on the div container used for images in the staging area, then override the sizing styles to the img itself). Seems simpler to just have an img here, but I can keep digging if you think it's worthwhile.

Reply:
Yep totally fine, that works!